examples/dendrogram.py

```diff
@@ -0,0 +1,97 @@
+import numpy as np
+import pandas as pd
+
+import neurom as nm
+from bluepysnap import Circuit
+from bluepysnap.sonata_constants import Edge
+from bluepysnap.bbp import Synapse
+
+from morph_tool.dendrogram import draw
+
+
+def plain_example():
```

Comment:
> Why not a symbolic link in the examples/ folder if we really keep the example in this folder.

Please elaborate here

Reply:
Now I see. It is possible to do with `plain_example` but it would be too much effort with `circuit_example`. The latter would require a proper test circuit which is too much. I just wanted to show with this example that it is possible to use `dendrogram` functionality with `bluepysnap.Circuit`.

Reply:
My proposition is to keep them at their existing place. Moving them to tests is excessive.

Reply:
Ok ok.